File: JAVA/Prime Number Checker/PrimeNumberChecker.java
import java.util.Scanner;

/**
 * Created by @amanmehara on 15/03/2017.
 */
public class PrimeNumberChecker {
    public static void main(String[] args) {
        System.out.println("isPrime? ");
        System.out.println();

        Scanner consoleInput = new Scanner(System.in);

        System.out.print("Number : ");
        long number = consoleInput.nextLong();
        System.out.println();

        boolean isPrime=true;

        //Prime number checker
        for (int i=2;i<=Math.sqrt(number);i++) {
            if (number % i == 0) {
                isPrime = false;
                break;
            }
        }

        if(isPrime) {
            System.out.println("Prime Number");
        }
        else {
            System.out.println("Not a Prime Number");
        }
        System.out.println();
    }
}

File: C/Prime Number Checker/primeNumberChecker.c
#include <math.h>
#include <stdio.h>

int main() {
	printf("Prime Number Checker \n");
	printf("\n");

	int number;

	printf("Enter a Natural Number (Greater than 1): \n");
	scanf("%d", &number);
	printf("\n");

	if (number < 2) {
		printf("Invalid Input \n");
		printf("\n");
		return 1;
	}

	for (int i = 2; i <= sqrt(number); i++) {
		if (!(number % i)) {
			printf("Number (%d) is Not a Prime Number \n", number);
			printf("\n");
			return 1;
		}
	}

	printf("Number (%d) is a Prime Number \n", number);
	printf("\n");

	return 0;
}